index.html

```diff
@@ -289,16 +283,14 @@ <h2 class="h3">Back to locations</h2>
 
             <!-- The hospital description panel -->
             <div class="fl-mapviz-hospital-panel" role="region" aria-live="polite">
-                <div class="fl-mapviz-expand-collapse-button-wrapper">
-                    <button type="button" class="fl-mapviz-expand-collapse-button" aria-expanded="true" aria-label="hospital panel">
-                        <svg class="fl-mapviz-collapse-button">
-                            <use xlink:href="#ArrowDown"></use>
-                        </svg>
-                        <svg class="fl-mapviz-expand-button">
-                            <use xlink:href="#ArrowUp"></use>
-                        </svg>
-                    </button>
-                </div>
+                <button type="button" class="fl-mapviz-expand-collapse-button" aria-expanded="true" aria-label="hospital panel">
+                    <svg class="fl-mapviz-collapse-button">
+                        <use xlink:href="#ArrowDown"></use>
+                    </svg>
+                    <svg class="fl-mapviz-expand-button">
+                        <use xlink:href="#ArrowUp"></use>
+                    </svg>
+                </button>
                 <div class="fl-mapviz-back-button-wrapper">
                     <button class="fl-mapviz-hospital-back-button" type="button">
                         <div class="fl-mapviz-icon-shadow">
```

Comment:
`npm run lint` shows two lint errors (seemingly contradictory):

> - index.html:
>   (466:-) Expected a newline at the end of the file.
>   (466:-) Unexpected additional newlines at the end of the file.
> 

These are side effects of the build process since the newline is there prior to the build.

Reply:
Thanks for the detailed explanation, @klown. I cannot reproduce the linting error. CI for this pull request also didn't hit the linting errors. Can you remove `/node_modules`, reinstall `npm i` and run `npm run lint` again? Please let me know how it goes. Thanks.

Reply:
@cindyli, after a closer look, the problem is caused by running `node buildSVG.js`.

Reply:
Oh, right. This is a known issue I looked into before. [This line](https://github.com/inclusive-design/covid-data-monitor/issues/33) is where empty lines are added. Unfortunately, I didn't find a solution for it.

I created https://github.com/inclusive-design/covid-data-monitor/issues/33 for it to be addressed separately.

Reply:
FYI: the "This line" link here goes to issue #33, not to any line of code.

Reply:
Right. My bad. The line of code that adds empty lines is [buildSVG.js#L34](https://github.com/inclusive-design/covid-data-monitor/blob/main/buildSVG.js#L34).